### PR TITLE
dnn/openvino: add log error message

### PR DESCRIPTION
**Pull request by Semmer2**

### CODE REVIEW

File: libavfilter/dnn/dnn_backend_openvino.h [hunk 1]
```diff
@@ -28,6 +28,21 @@
...
+} OVContext;
+
+typedef struct OVModel{
+    OVContext ctx;
+    ie_core_t *core;
+    ie_network_t *network;
+    ie_executable_network_t *exe_network;
+    ie_infer_request_t *infer_request;
+    ie_blob_t *input_blob;
+} OVModel;
```
Comment (guoyejun) [inline libavfilter/dnn/dnn_backend_openvino.h:45]: why  we need to define ovcontext and ovmodel in this .h file?

Comment (Semmer2) [reply libavfilter/dnn/dnn_backend_openvino.h:45]: Well, I think all the define of struct should be in header file, just like the define of NativeModel & NativeContext in dnn_backend_native.h. Would this be the reason why all CI failed?

Comment (guoyejun) [reply libavfilter/dnn/dnn_backend_openvino.h:45]: no, we should put only necessary things in header file.

looks it has no relation with ci failure.

Comment (Semmer2) [reply libavfilter/dnn/dnn_backend_openvino.h:45]: OK, got you, I will modify this part in next version.

**Pull request closed without merging**